kedro_neptune/__init__.py

```diff
@@ -80,6 +83,7 @@ def neptune_commands():
 #GLOBAL CONFIG
```

Comment:
I think that we could keep this config in separate file.
It'll be more PRO solution ;)

Reply:
It's not config (in the general meaning). It's template of yaml file. As separate file (without `.py` extension) it should be placed as package files but I don't know if Kedro will handle it properly. What about moving all templates to `template.py`?

Reply:
Ok, I remember now...
It's the code which generates some project.
Yes, we could move those variables outside, but at this point let keep it as it is. There's no so much text.